.travis.yml

```
@@ -0,0 +1,13 @@
+
+sudo: required
+dist: xenial
+language: python2.7
+
+services:
+  - docker
+
+script:
+  - cd tools/docker/build
+  - sudo ./image-build.sh
+  - sudo ./build.sh maptools.py build map -n MSGLIB=zmq BUILD_TESTS=ON
```

```suggestion
  - sudo ./build.sh  map -n MSGLIB=zmq BUILD_TESTS=ON
```
The `maptools.py` and `build` arguments are passed by the ENTRYPOINT definition in the docker.